packages/date-picker/test/form-input.test.js

```diff
@@ -16,6 +16,48 @@ customElements.define('vaadin-date-picker-2016', DatePicker2016);
 describe('form input', () => {
   let datePicker;
 
+  describe('initial', () => {
+    let validateSpy;
+
+    beforeEach(() => {
+      datePicker = document.createElement('vaadin-date-picker');
+      validateSpy = sinon.spy(datePicker, 'validate');
+    });
+
+    afterEach(() => {
+      datePicker.remove();
+    });
+
+    it('should not validate by default', async () => {
+      document.body.appendChild(datePicker);
+      await nextRender();
+      expect(validateSpy.called).to.be.false;
+    });
+
+    it('should not validate when the field has an initial value', async () => {
```

Comment:
There is no test for initial value and `invalid` - is that intended?

Reply:
Good question. There should be such a test case indeed. 

Furthermore, I think I  was hasty in preventing the initial validation also in the case `min` and `max` are set. It seems only reasonable to prevent the initial validation when there are no constraints specified. The developer should not expect that manual managing of `invalid` would work with constraints.